modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/LogArea.scala

```diff
@@ -167,28 +172,47 @@ object LogArea {
   private val component = ScalaComponent.builder[Props]("LogArea")
     .initialState(State.Zero)
     .renderPS { ($, p, s) =>
+      val toggleIcon = (p.log().display === SectionOpen).fold(IconAngleDoubleDown, IconAngleDoubleUp)
+      val toggleText = (p.log().display === SectionOpen).fold("Hide Log", "Show Log")
       <.div(
         ^.cls := "ui sixteen wide column",
         SeqexecStyles.logSegment,
         <.div(
           ^.cls := "ui secondary segment",
+          SeqexecStyles.logSecondarySegment,
           <.div(
-            ^.cls := "ui form",
+            ^.cls := "ui grid",
             <.div(
-              ^.cls := "fields",
-              SeqexecStyles.selectorFields,
-              s.selectedLevels.map {
-                case (l, s) =>
+              ^.cls := "ui row",
+              SeqexecStyles.logControlRow,
+              <.div(
+                ^.cls := "ui six wide column",
+                SeqexecStyles.logVisibilityField,
+                Button(Button.Props(icon = Option(toggleIcon), labeled = true, compact = true, size = SSize.Small, onClick = p.log.dispatchCB(ToggleLogArea)), toggleText)
```

Comment:
What does the new property `compact` mean / do?

Reply:
It makes the button slightly shorter:
https://semantic-ui.com/elements/button.html#compact